### DISCUSSION TITLE
New Slogan

### DISCUSSION THREAD

**Discussion by jacoobes** [sern-handler/website#53] [Ideas]
# closed for suggestions, keeping this for historical purposes

Our current slogan and frontpage eye catcher is VERY vague.
![image](https://github.com/sern-handler/website/assets/76754747/0161c1fe-7617-46ce-ae34-f7343047cc9b)
Per community discussion, what are we handling? It's not very clear.
I wouldn't even know myself.
### Some proposals
- Your Discord bot handled the now way
- Our Expertise, your discord bots
- The 21st century Discord bot ecosystem 
- Create discord bots, now
- Your Discord bot, less consern :kekw:

**Comment by jacoobes** [comment]
hi

**Reply by SrIzan10** [reply]
hi seren

**Comment by jacoobes** [comment]
fun fact, our slogan was ripped from vscode

![image](https://github.com/sern-handler/website/assets/76754747/6dada46e-d8ab-441a-a89f-b53392cc5447)

**Reply by SrIzan10** [reply]
it was a placeholder and we just forgot about it

**Comment by SrIzan10** [comment]
![image](https://github.com/sern-handler/website/assets/66965250/4ff64e1e-5c41-49be-9a43-21dc1ee0a266)
what about catching the user's attention?

**Reply by jacoobes** [reply]
somethign like this is can be really effective
